unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/WelcomeHUD/WelcomeHUDController.cs

```diff
@@ -1,20 +1,39 @@
+using System;
 using DCL.Helpers;
 using DCL.Interface;
 using UnityEngine;
 
+[Serializable]
+public class MessageOfTheDayConfig
```

Comment:
Lately we are adding an overview description for all new classes we implement (also for the public methods that we consider necessary) in order to keep a basic documentation of each one. It would be fine to add it to this new HUD.

Reply:
Oh, I had no idea :o